georocket-server/integration-tests/src/berlin_alexanderplatz_mini.xml

```diff
@@ -0,0 +1,571 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
These files should already be in the `integration-tests-parent-image`:
https://hub.docker.com/r/georocket/integration-tests-parent-image/

This is the image the integrations tests run in.